src/main/scala/LarParser.scala

```diff
@@ -9,15 +9,36 @@ trait LarParser {
   def parseLar(s: String): List[String] = {
     val values = s.split('|').map(_.trim)
     var errors = List[String]()
-    val intFields = List(0, 2, 5, 6, 7, 8, 9, 10, 11, 12, 17, 18, 24, 29, 30, 32, 37, 38)
+    val intFields = Map(
```

Comment:
I just changed it to numericFields because that is I believe what drives the classification. Is that clearer?